### PR TITLE
Code Readability - Weapons

### PR DESCRIPTION
**Pull request by KalevTait**
## What Does This PR Do
* Converts 0/1 to FALSE/TRUE where those #defines make the code more readable. Note: There are a few cases where this converts 0.0/1.0 to FALSE/TRUE or null to FALSE - this change is consistent with how the variable are used.
* Simplifies some if statements, either by removing unnecesary == or by removing to many brackets

## Why It's Good For The Game
More readable code is easier to work with.

### CODE REVIEW

File: code/game/objects/items/weapons/grenades/ghettobomb.dm [hunk 1]
```diff
@@ -11,9 +11,9 @@
 	throw_range = 7
 	flags = CONDUCT
 	slot_flags = SLOT_BELT
-	active = 0
+	active = FALSE
 	det_time = 50
```
Comment (hal9000PR) [inline code/game/objects/items/weapons/grenades/ghettobomb.dm:15]:
```suggestion
	det_time = 5 SECONDS
```

**Comment by AffectedArc07** [comment]
DNM as this will be a heavy conflict to the sprites PR

File: code/game/objects/items/weapons/teleportation.dm [hunk 2]
```diff
@@ -127,7 +127,7 @@ Frequency:
 		if(T.x>world.maxx-8 || T.x<8)	continue	//putting them at the edge is dumb
 		if(T.y>world.maxy-8 || T.y<8)	continue
 		A = get_area(T)
-		if(A.tele_proof == 1) continue // Telescience-proofed areas require a beacon.
+		if(A.tele_proof) continue // Telescience-proofed areas require a beacon.
```
Comment (hal9000PR) [inline code/game/objects/items/weapons/teleportation.dm:130]: would you make this indent properly?

**Review by hal9000PR — CHANGES_REQUESTED** [review]
other than this LGTM

File: code/game/objects/items/weapons/teleportation.dm (hunk 2 above)
Comment (KalevTait) [reply code/game/objects/items/weapons/teleportation.dm:130]: I'm unsure what you are refering to? Do you mean indent the continue to line up with the ones above, or the comment, or both?

Comment (KalevTait) [reply code/game/objects/items/weapons/teleportation.dm]: Finally figured out what you meant by this ;p

**Review by hal9000PR — APPROVED** [review]

**Pull request merged**